tests/test_create_reports.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+import numpy as np
+from tinnsleep.data import CreateRaw
+from tinnsleep.create_reports import  preprocess, reporting
+import numpy.testing as npt
+from tinnsleep.utils import epoch
+
+
+def test_preprocess():
+    np.random.seed(42)
+    data = np.random.randn(4, 400)
+
+    data[0] = 1e-7 * data[0]
+    data[1] = 1e-7 * data[1]
+    data[1][100:200] += 100
```

Comment:
Here is a catch:
`_is_good()` uses max-max thresholding.

Therefore I advice instead to do:
`data[1][100] += 100` if you want to detect artifact
`data[1][100:200] = 100` if you want to detect flat electrode

I hope it helps ;)

Reply:
I've made it work with:
`data[1][110:120] = 1000 * data[1][110:120]`
To make it detect an artefact, that is present on only the epoch from 100:150. I tried with a factor of 1e4, but it began to influence surronding epochs (the one before and after) highlighting the limit of convergence of the filter.